recipe/meta.yaml

```diff
@@ -42,10 +49,16 @@ test:
     - if not exist %LIBRARY_LIB%\armadillo.lib exit 1  # [win]
```

Comment:
Will it build on Windows static library, right? https://gitlab.com/conradsnicta/armadillo-code/-/blob/12.6.x/CMakeLists.txt?ref_type=heads#L181

Reply:
or is it just import library ?
https://learn.microsoft.com/en-us/windows/win32/dlls/dynamic-link-library-creation#creating-an-import-library

Reply:
Good catch. I can only see `armadillo.lib` in the log for `win` https://cloud.prefect.io/anaconda/flow-run/f2d6d989-07e1-4972-af3b-42cf95f92b37

also: 
https://gitlab.com/conradsnicta/armadillo-code/-/blob/12.6.x/CMakeLists.txt?ref_type=heads#L223-228

```
if(MSVC AND BUILD_SHARED_LIBS)
  message(STATUS "")
  message(STATUS "*** WARNING: building shared library with MSVC is not supported." )
  message(STATUS "*** WARNING: if anything breaks, you get to keep all the pieces.")
  message(STATUS "")
endif()
```

Reply:
Yes, it defaults to static and I don't want to build in shared because of that warning.